src/main/java/api_data/FXData.java

```diff
@@ -0,0 +1,33 @@
+package api_data;
+
+import api_connection.FXDailyService;
+import api_connection.FXDailyServiceFactory;
+import api_connection.TimeSeries;
+import com.google.gson.internal.LinkedTreeMap;
+import retrofit2.Response;
+
+import java.io.IOException;
+
+public class FXData {
+    private TimeSeries timeSeries = null;
+
+    /* ticker should be formatted as "ABC" */
+    public FXData(String ticker) {
+        FXDailyServiceFactory factory = new FXDailyServiceFactory();
+        FXDailyService service = factory.getInstance();
+
+        try {
+            Response<TimeSeries> response = service.getFXData(ticker).execute();
+            timeSeries = response.body();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+    }
+
+    //format date as "YYYY-MM-DD"
```

Comment:
Could we change this to a LocalDate? That seems to be the class we're using throughout for dates.

Reply:
this is the format thats used in the api. we could do the reformatting here or before it gets passed in

Reply:
Ok, I guess we can decide when we see how it's being integrated.